### ISSUE TITLE
Thinking of moving VsVim to a GitHub org

### ISSUE THREAD

**Issue by jaredpar** [VsVim/VsVim#2485]
Given there are more people participating in the project it feels weird, to me at least, that this is still listed under my personal account. Most longer lived OSS projects end up creating an org and putting the project there. Been considering doing that lately and have already pre-reserved the organization: https://github.com/VsVim

Wanted to see if there were any objections before I made this move though. GitHub supports moving projects into orgs so it should be fairly smooth.

**Comment by ricksladkey** [comment]
I think that's a great idea for many reasons, but perhaps the biggest one I can think of is that the organization's GitHub pages page would be https://vsvim.github.io, which puts the hub in GitHub for VsVim. Not much needs to go there right away, but having it is much more accessible to non-programmers than, say, https://github.com/jaredpar/VsVim is.

**Comment by jaredpar** [comment]
That's a good point. I hadn't thought of that. 

I'm on vacation for the next week. Once I get back I'm going to make the switch over to an organization. Didn't want to do it on the way out just in case I messed something up 😄

**Comment by jaredpar** [comment]
@ricksladkey reading through the docs the one potential disruption the transfer could have is clearing issue assignments. When transferring between a user and an organization issue assignments are cleared unless the assignee is a member of the organization. 

Sent you an invitation ~1 week ago. It's possible it ended up in your spam account though (most invites do for me). Believe you can accept by just visiting the org page directly: https://github.com/VsVim

If you don't want to be an organization member that's fine. But I didn't want all your issues to get unassigned to be a surprise. Not sure why this rule happens given you maintain your access as part of teh migration but it is called out in the notes.

**Comment by ricksladkey** [comment]
Ha, ha, I totally want to be in the organization! Just accepted. So now I am.

As for why I didn't see the invitation. the most likely explanation would generally be that I just overlooked it. But it turns out it's my own fault for another reason: years ago I created a VsVim sub-folder in my mail and auto-filed VsVim-related GitHub email in it. I no longer use sub-folders this way.

Anyway, it's all fixed now.
